docs/articles/nunit/writing-tests/attributes/parallelizable.md

```diff
@@ -4,11 +4,20 @@ uid: parallelizableattribute
 
 # Parallelizable
 
+Added in **NUnit 3.7.**
```

Comment:
```suggestion
Added in NUnit `3.7.x`
```
⚠️ This looks like it will appear as a bolded "3.7." rather than `3.7.x`.

Reply:
I meant `3.7` with a period after it. Backticks here also doesn't match the form I used in `fixturelifecycle.md`. Let's pick one of the two. Either bold or back ticks, I don't  think we need the `.x` as features are introduced in 2nd digit releases. We only issue a third digit release for hotfixes.